### PR TITLE
PR, bug and feature templates added

### PR DESCRIPTION
**Pull request by jpontongradiant**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bugreport.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+name: Bug Report
+description: File a bug report or issue
+title: "[Bug]: "
+body:
+  - type: markdown
+    attributes:
+      value: > 
+        **Please note**
+        
+        This form should only be used if _you can reporoduce_ the bug the in the latest version of openverso-images.
```
Comment (avrodriguezgrad) [inline .github/ISSUE_TEMPLATE/bugreport.yaml:10]: Change to 5g-images

File: .github/ISSUE_TEMPLATE/bugreport.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        
+        **This form is not for support requests.*
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: >
+        Please describe in detail the steps needed to reproduce this bug. These steps
+        should reproduce the issue on the most current release of openverso-images. Be sure to
```
Comment (avrodriguezgrad) [inline .github/ISSUE_TEMPLATE/bugreport.yaml:18]: Change to 5g-images

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+name: Feature request
+description: Propose an enhancement to openverso-images
```
Comment (avrodriguezgrad) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:2]: Change to 5g-images

**Review by avrodriguezgrad — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bugreport.yaml (hunk 2 above)
Comment (jpontongradiant) [reply .github/ISSUE_TEMPLATE/bugreport.yaml]: Done

**Review by avrodriguezgrad — APPROVED** [review]

**Pull request merged**